packages/twenty-server/src/engine/workspace-manager/workspace-migration-runner/services/workspace-migration-enum.service.ts

```diff
@@ -214,4 +224,26 @@ export class WorkspaceMigrationEnumService {
       RENAME TO "${newEnumTypeName}"
     `);
   }
+
+  private async getEnumTypeName(
+    queryRunner: QueryRunner,
+    schemaName: string,
+    tableName: string,
+    columnName: string,
+  ): Promise<string> {
+    const result = await queryRunner.query(
```

Comment:
Instead of recomputing the enum name via the column only, since the table can also change we need to retrieve the exact name from the catalogue

Reply:
good as there already are renamed tables in production for which we did not update the enum